src/views/webview-app/components/overview-page/overview-header.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { ActionTypes, LinkClickedAction } from '../../store/actions';
+
+const styles = require('./overview-page.less');
+
+type dispatchProps = {
+  onLinkClicked: (screen: string, linkId: string) => void;
+};
+
+type props = dispatchProps;
+
+class OverviewHeader extends React.Component<props> {
+  onLinkClicked = (screen: string, linkId: string): void => {
+    this.props.onLinkClicked(screen, linkId);
+  };
+
+  renderMongoLogo(): React.ReactNode {
+    return (
+      <svg width="188" height="50" viewBox="0 0 188 50" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could we put this in a separate file, either with a 👿  webpack 👿  loader, or just as a separate component for the logo?

Reply:
Just noticed that we need classes there, so probably not a webpack loader :)

Reply:
good call, updated